### DISCUSSION TITLE
Does Report Portal have the ability to show results from Cucumber hooks?

### DISCUSSION THREAD

**Discussion by grant-mccarriagher** [reportportal/reportportal#1968] [General]
Our suite uses Webdriverio, CucumberJS, and the wdio-reportportal-reporter plugin to report data to Report Portal. We've noticed that hook failures don't display on test results (it just shows all the steps as failed without any logging), and are trying to figure out which components in our testing suite aren't supporting this (WDIO? CucumberJS? the third party RP reporter plugin? Report Portal itself?).

Anybody else run into this issue with their own test suites? Thanks!

**Comment by DzmitryHumianiuk** [comment]
@grant-mccarriagher, at this point, ReportPortal is entirely driven by the way test reporters send data to it. 
It reflects the data based on how WDIO processes the case, and how it's handled by cucumberJS if that's involved in the structure generation. 
You should then examine how the data is transmitted via CucuJS -> WDIO -> and RP reporter plugin.

Keep in mind that JS reporters can be unpredictable sometimes, and they often don't follow common patterns in reporting and event sequences. We've observed this behavior across the vast majority of JS runners.
